python/ray/serve/batching.py

```diff
@@ -304,6 +318,47 @@ def get_max_batch_size(self) -> int:
     def get_batch_wait_timeout_s(self) -> float:
         return self.batch_wait_timeout_s
 
+    def _get_curr_iteration_start_time(self) -> float:
+        """Gets current iteration's start time on default _BatchQueue implementation.
+
+        Returns -1 if the batch handler doesn't use a default _BatchQueue.
+        """
+
+        if hasattr(self.queue, "curr_iteration_start_time"):
+            return self.queue.curr_iteration_start_time
+        else:
+            return -1
+
+    async def _is_batching_task_alive(self) -> bool:
+        """Gets whether default _BatchQueue's background task is alive.
+
+        Returns False if the batch handler doesn't use a default _BatchQueue.
+        """
+
+        if hasattr(self.queue, "_handle_batch_task"):
+            if self.queue._handle_batch_task is None:
+                return False
+            else:
+                _, pending = await asyncio.wait(
+                    [self.queue._handle_batch_task], timeout=0
+                )
+                return len(pending) == 1
+        else:
+            return False
```

Comment:
I think this is just: `not self.queue._handle_batch_task.done()`

Reply:
I don't think this method is actually useful if we have a top-level `try: except` block though, would suggest to remove it

Reply:
I agree in theory, but I think it'll help us sanity check that the failures in Aviary aren't caused by the task failing. Since it's a private method, I see no harm in keeping it around until we debug the failures.